docs/configuration/configuration.md

```diff
@@ -397,6 +412,9 @@ tags:
 # The time after which the provided names are refreshed.
 # On large setup it might be a good idea to increase this value because the catalog will change all the time.
 [ refresh_interval: <duration> | default = 30s ]
+
+# Wether to generate a target per exporsed path if the discovered service is connect proxy.
+[ fetch_exposed_paths: <boolean> | default = false]
```

Comment:
Should we take a `role` approach, with one role being `services` and one role being `exposed_path` ?

Reply:
I'm not an expert on the prometheus side, but I see most users using both options most of the time. If we use a role approximation most users that will want to use the exposed path role would also use the services, that's why I set it as a configuration option instead of a role, but maybe users would use several scrape configurations, in one setting the role to service and in another setting it to expose_path, so I really don't know which is the best option here. 

If we want to get the exposed paths of one service i guess that we don't need that service, but maybe we want other services that don't expose paths.

TL;DR: Both options will work, don't know which one is better

Reply:
In case we have another great feature like this or thing to extract from consul in the future, I say let's have a `role:` approach.